### PR TITLE
[issue #4074][pulsar-io]Update document of PostgreSQL for debezium

### PR DESCRIPTION
**Pull request by tuteng**
Fixes #https://github.com/apache/pulsar/issues/4074

Master Issue: #4074 

### Motivation
There is currently a lack of documentation on PostgreSQL. Therefore, some useful documents about PostgreSQL are added.

### Modifications
Update document of PostgreSQL for debezium

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API: (yes / no)
  - The schema: (yes / no / don't know)
  - The default values of configurations: (yes / no)
  - The wire protocol: (yes / no)
  - The rest endpoints: (yes / no)
  - The admin cli options: (yes / no)
  - Anything that affects deployment: (yes / no / don't know)

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@sijie Glad to help.

File: site2/docs/io-cdc-debezium.md [hunk 1]
```diff
@@ -26,56 +26,44 @@ The Configuration is mostly related to Debezium task config, besides this we sho
 | `pulsar.service.url` | `true` | `null` | Pulsar cluster service url. |
 | `offset.storage.topic` | `true` | `null` | Record the last committed offsets that the connector successfully completed. |
 
-### Configuration Example
+### Configuration Example from MySql
```
[the remark this answers is not in the document]
Comment (Anonymitaet) [reply site2/docs/io-cdc-debezium.md]: Adjust the structure as follows:

```
## Source Configuration Options

## Example of MySQL
### Configuration
### Usage

## Example of PostgreSQL
### Configuration
### Usage
```

File: site2/docs/io-cdc-debezium.md [hunk 2]
```diff
@@ -135,4 +129,106 @@ mysql> UPDATE products SET name='1111111111' WHERE id=101;
...
+ 106 | hammer             | 16oz carpenter's hammer                                 |      1
+ 107 | rocks              | box of assorted rocks                                   |    5.3
+ 108 | jacket             | water resistent black wind breaker                      |    0.1
+ 109 | spare tire         | 24 inch spare tire                                      |   22.2
+ 101 | 1111111111         | Small 2-wheel scooter                                   |   3.14
+(9 rows)
+
+postgres=# UPDATE products SET name='1111111111' WHERE id=107;
+UPDATE 1
+```
```
Comment (Anonymitaet) [inline site2/docs/io-cdc-debezium.md]: Need to add the step of verifying the change will be captured in the topic?

**Comment by tuteng** [comment]
run java8 Tests

**Comment by tuteng** [comment]
run java8 Tests

**Comment by tuteng** [comment]
run java8 Tests

**Comment by sijie** [comment]
@Anonymitaet can you please review this pull request again? since @tuteng has incorporated your review comments.

**Comment by Anonymitaet** [comment]
@sijie Sorry, just saw this message because I was overwhelmed by review emails.
LGTM.

**Pull request merged**